contracts/LeoToken.sol

```diff
@@ -0,0 +1,69 @@
+//SPDX-License-Identifier: Unlicensed
+pragma solidity ^0.8.0;
+
+import 'hardhat/console.sol';
```

Comment:
Tego przed released warto się pozbyć. To nie release, ale wiadomo :D

Reply:
Ups 😅